.buildkite/pipeline.yaml

```diff
@@ -0,0 +1,30 @@
+steps:
+  - command:
+    - "./build.sh"
```

Comment:
Do we care about having the binaries built? If not, then we should use `go build ./cmd/...`, if yes, then we should expose them as artifacts.

Reply:
I don't think we care at this point.